CHANGELOG.md

```diff
@@ -11,8 +11,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - A default role `Vulnerability Manager` has been added that provides sufficient privileges to analyze and manage system vulnerabilities.
 - A default role `Network Graph Viewer` has been added that provides sufficient privileges to display network graphs.
 - A new command `roxctl central login` has been added that allows to use a user's token within roxctl instead of an API token or admin password.
+- ROX-16696: A new DelegatedRegistryConfig API at `/v1/delegatedregistryconfig` has been added that provides dynamic configuration for local registry scanning (replaces `ROX_FORCE_LOCAL_IMAGE_SCANNING`).
```

Comment:
Based on these CHANGELOG entries, does the ticket number in the PR title make sense still?

Reply:
Good catch, because this particular change covers multiple stories, will update `CHANGELOG.md` that use `ROX-15447`. The ticket in the PR title is 'more accurate' in this case.